book/website/figures/_figure-list.md

```diff
@@ -23,8 +23,11 @@
 | eyeball_test_2             | Testing                   | Results tested by seeing if they 'look right'     |
 | eyeball_test_3             | Testing                   | Results tested by seeing if they 'look right'     |
 | eyeball_test_error         | Testing                   | Bug detected by result 'looking wrong'            |
+| first-pull-request         | Style guide               | Two people making a first pull request on GitHub |
```

Comment:
I'm changing the filenames to be underscored, maybe you want to update them while in the PR (I'll add an action for it to automate it so no big deal to not do it here)

Reply:
Fixed! Thank you @bsipocz !